Build printable version without background (#553)
Heegu-sama/Homm3BG: README.md

            <td align="center">🖨️<br>Printable</td>
            <td align="center">🖥️<br>Digital</td>
            <td align="center">🖨️<br>Printable</td>

>             <td align="center">🖨️<br>Printable<br><i>(without art backgrounds)</i></td>

Reviewer: ```suggestion
            <td align="center">🤑<br>Economy Printable</td>
```
Let's get creative with emojis ;-) The explanation is below the table anyway.

            <td align="center"><a href="https://github.com/Heegu-sama/Homm3BG/releases/download/v1.2/Heroes3_Ukrainska_Rules_Rewrite_12_Printable.pdf">завантажити</a></td>
            <td align="center"><a href="https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ua/main_ua.pdf">️завантажити</a></td>
            <td align="center"><a href="https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ua/printable_ua.pdf">завантажити</a></td>

>             <td align="center"><a href="https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/en/economy_printable_ua.pdf">завантажити</a></td>

Reviewer 2: en/ua?

Same in other rows here

Author: Thanks. Fixed

Maybe we don't need to have languge in path and the name but let's leave it like this for now. This PR is large enough :)

Reviewer: We need them, because they are the git branch that the files are pushed to in another repo. If you have 7 workflows force pushing 3 blobs at the same time, you need 7 different branches.

🖥️ The digital build is intended to be read on electronic devices.
It has convenient hyperlinks that navigate you around the text.<br>
🖨️ The printable build appends page numbers to select clickable hyperlinks, includes an index page at the end, and makes sure the document has 56 pages, and uses CMYK to be print friendly.
Best served for those who would like to have it on paper 🤞<br>

> 🖨️ The printable build **without art backgrounds** is the same as the regular printable build but without the backgrounds as the name suggests. This is for people who want to save their toners or don't have printers that can print edge to edge where the white borders around the art backgrounds don't look that good.

Reviewer: ```suggestion
🤑 The economy printable build has **no art backgrounds**. Other than that, it's the same as the regular printable build.
This is for people who want to save their toners or don't have printers that can print edge to edge where the white borders around the art backgrounds don't look that good.
```

 - uses [CMYK](https://en.wikipedia.org/wiki/CMYK_color_model) (experimental) 🎨

all of which to be print friendly.
Best served for those who would like to have it on paper 🤞<br>
🤑 The economy printable build has **no art backgrounds**. Other than that, it's the same as the regular printable build.

> This is for people who want to save their toners or don't have printers that can print edge to edge where the white borders around the art backgrounds don't look that good.

Reviewer 2: we should make printable build (background or not) to use crop and not a4 directly; but that's out of scope for this change
